src/worker/wrapTryCatchAroundAction.js

```diff
@@ -1,3 +1,37 @@
+import formatString from "../helpers/formatString";
+
+/**
+ * Logs and formats the error message to send back to the plugin
+ * when an analysis web worker action fails.
+ *
+ * @param {Logger}	logger					The logger instance to log with.
+ * @param {Error} 	error					The error to log.
+ * @param {Object}	payload					The action payload.
+ * @param {string} 	[errorMessagePrefix=""]	The prefix of the error message.
+ *
+ * @returns {string} the error message to send back.
+ */
+const handleError = function( logger, error, payload, errorMessagePrefix = "" ) {
```

Comment:
I presume you moved this because of the complexity. However, CodeClimate now says the following:
```
Function `handleError` has a Cognitive Complexity of 6 (exceeds 5 allowed). Consider refactoring.
```

Do you see any other improvements to reduce complexity?